src/Console/ModelsCommand.php

```diff
@@ -431,7 +432,7 @@ protected function getPropertiesFromMethods($model)
                         $args = $this->getParameters($reflection);
                         //Remove the first ($query) argument
                         array_shift($args);
-                        $this->setMethod($name, '\Illuminate\Database\Eloquent\Builder|\\' . $reflection->class, $args);
+                        $this->setMethod($name, '\Illuminate\Database\Eloquent\Builder|\\' . $reflection->class, $args, 'static');
```

Comment:
@bastien-phi maybe `phpcs` warnings. This line exceeds 120 characters.
https://travis-ci.org/barryvdh/laravel-ide-helper/jobs/261771638#L347